draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -982,8 +982,17 @@ ExternalMu-ML-DSA.Sign(sk, mu):
   sigma = ExternalMu-ML-DSA.Sign_internal(sk, mu, rnd)
```

Comment:
Would need to update this line to:
`sigma = ExternalMu-ML-DSA.Sign_internal(sk, mu, rnd, isExternalMu=true)`